finetuner/experiment.py

```diff
@@ -237,7 +238,8 @@ def _create_config_for_run(
         ]
         public = kwargs[PUBLIC] if kwargs.get(PUBLIC) else False
         model = config.ModelConfig(
-            name=model,
+            name=model if not kwargs.get(MODEL_ARTIFACT) else None,
+            artifact=kwargs.get(MODEL_ARTIFACT),
```

Comment:
Just to make sure I understand, this means that if there is a `MODEL_ARTIFACT`, then `name` is not set?

Reply:
yes, because George implemented the jsonschema in a way that both arguments are mutually exclusive, which makes somehow sense, because otherwise it would be a little bit hard to see what will happen, e.g., if a new model is constructed or the artifact is used. So in this way, the final config to be send to the API is more clear on what a run should do.